arxivdigest/api/README.md

```diff
@@ -81,6 +81,7 @@ Fields returned for each user:
   - `homepages`: list of homepages
   - `keywords`: list of self-supplied keywords
   - `categories` : list of arXiv categories user is interested in
+  - `company`: the company the user registered with
```

Comment:
we should call it `organization` everywhere (it's not necessarily a commercial entity, often it'll be a university)